src/main/plugins/org.dita.html5/xsl/hazard-d.xsl

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xsl:stylesheet version="2.0"
+     xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+     xmlns:ditamsg="http://dita-ot.sourceforge.net/ns/200704/ditamsg"
+     xmlns:dita-ot="http://dita-ot.sourceforge.net/ns/201007/dita-ot"
+     xmlns:xs="http://www.w3.org/2001/XMLSchema"
+     exclude-result-prefixes="ditamsg dita-ot xs">
+  
+  <xsl:param name="default.hazard.image.filename" as="xs:string" select="'ISO_7010_W001.svg'"/>
+  <xsl:variable name="default.hazard.image" as="xs:string">
+    <xsl:sequence select="concat(/processing-instruction()[name()='path2rootmap-uri'], $default.hazard.image.filename)"/>
+  </xsl:variable>
+  
+  <xsl:template match="*[contains(@class,' hazard-d/messagepanel ')]" mode="get-element-ancestry"><xsl:value-of select="name()"/></xsl:template>
+  <xsl:template match="*[contains(@class,' hazard-d/messagepanel ')]/*" mode="get-element-ancestry"><xsl:value-of select="name()"/></xsl:template>
+  <xsl:template match="*[contains(@class,' hazard-d/hazardstatement ')]">
+    <xsl:variable name="type" select="(@type, 'caution')[1]" as="xs:string"/>
+    <xsl:variable name="hazardcolor">
```

Comment:
I think this should be moved to CSS. Then the hazard statement will not be output with background colour if CSS is not loaded, but I think the image should be enough.

Reply:
:+1: forgot I'd started out that way, doing this in a plugin where I couldn't depend on the styles being in the default CSS. Definitely better to move.

Reply:
Maybe generate `class="hazardstatement--{$type}"` and add matching styles to SCSS.